src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -59,6 +59,32 @@
         az security alert update -g "myRg" --location "centralus" -n "alertName" --status "activate"
 """
 
+helps['security atp'] = """
+type: group
+short-summary: View and manage advanced threat protection settings.
+"""
+
+helps['security atp show'] = """
```

Comment:
Are you sure about `show` for the command name? Maybe `get` is better suited. I might be wrong though

Reply:
`show` is a general Azure CLI naming convention.